src/main/java/com/rakovets/course/java/core/practice/arrays/Task03.java

```diff
@@ -45,9 +65,16 @@ static double[] getAverageMarks(int[][] marks) {
      * @return минимальная отметка
      */
     static int[] getMinMarks(int[][] marks) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int minMark[] = new int[marks.length];
+        for (int i = 0; i < marks.length; i++) {
+            int minMarkInRow = marks[i][0];
+            for (int j = 0; j < marks[0].length; j++) {
+                if (minMarkInRow > marks[i][j]) {
+                    minMarkInRow = marks[i][j];
+                }
+            } minMark[i] = minMarkInRow;
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"